assets/src/edit-story/components/panels/stylePreset/header.js

```diff
@@ -80,36 +107,35 @@ function PresetsHeader({
   const hasPresets = colors.length > 0;
 
   const getActions = () => {
-    return !isEditMode ? (
+    return (
       <>
         {hasPresets && (
-          <EditModeButton
+          <Button
+            Icon={EditMode}
             onClick={(evt) => {
               evt.stopPropagation();
-              setIsEditMode(true);
+              setIsEditMode(!isEditMode);
             }}
-            aria-label={__('Edit presets', 'web-stories')}
+            aria-label={
+              isEditMode
+                ? __('Exit edit mode', 'web-stories')
+                : __('Edit presets', 'web-stories')
+            }
+            isEditMode={isEditMode}
+          >
+            {isEditMode ? __('Exit', 'web-stories') : <EditPencil />}
+          </Button>
+        )}
+        {!isEditMode && (
+          <Button
+            Icon={AddColorPresetButton}
+            onClick={handleAddColorPreset}
+            aria-label={__('Add preset', 'web-stories')}
           >
-            <EditPencil />
-          </EditModeButton>
+            <AddPage />
```

Comment:
`AddPage`? Let's rename to something more accurate?

Reply:
Didn't even consider that, but it makes sense. Will look into it.